src/components/Home.js

```diff
@@ -0,0 +1,23 @@
+import React, { useEffect } from 'react';
+import { useHistory } from 'react-router-dom';
+
+const Home = ({ token, handleTokenCreation }) => {
+  const history = useHistory();
+
+  useEffect(() => {
+    if (token) {
+      history.push('/list-view');
+    }
+    localStorage.setItem('token', token);
+  }, [token, history]);
+
+  return (
+    <main>
+      <button onClick={handleTokenCreation} className="button__home">
+        Create a Shopping List
+      </button>
+    </main>
+  );
+};
+
+export default Home;
```

Comment:
Great idea to create a home component 🙌🏽 On the home view, only the button should be shown. For a user, it should not be possible to switch between the list view and the add item view from the home component.

Reply:
Hi @YuriDevAT, the navlinks have been removed